src/run.py

```diff
@@ -35,7 +35,7 @@ class TemplateContainer:
     file_name: str
     output_dir: Path
     active: bool
-    arguments: dict[str, str]
+    arguments: dict[str, str] | None
```

Comment:
might as well default this to None and remove the argument everywhere where it is currently set to `None`
```suggestion
    arguments: dict[str, str] | None = None
```

Reply:
Makes sense! Done.